plotting_tools/plot_Rlxy.C

```diff
@@ -0,0 +1,253 @@
+#include <filesystem>
+#include "PlotHelpers.h"
+
+void plot_Rlxy()
+{
+  // Options for plotting:
+  bool weighted = true;
+  bool overflow = true;
+  int int_lumi = 150e3; // pb-1
+
+  string username = getenv("USER");
+
+  string base_path; 
+  string base_path_signal =  "/nfs/dust/cms/user/" + username + "/ttalps/signals_ctau-default_new-dimuon-mass-cuts/hists/";
+  string base_path_background =  "/nfs/dust/cms/user/" + username + "/ttalps/backgrounds_new-dimuon-mass-cuts/hists/";
+  string output_path;
+  if(username == "lrygaard") {output_path = "/afs/desy.de/user/l/lrygaard/ALPpheno/plots_Rlxy/";}
+  // else if(username == "jniedzie") {output_path = "/Users/jeremi/Documents/Physics/DESY/ttalps/data.nosync/hists/";}
+  // Only set up for user lrygaard now
+  else {cout << "Error: unrecognized user." << endl;}
+
+  filesystem::path p(output_path);
+  if(!filesystem::exists(p))
+  {
+    filesystem::create_directory(p);
+    p = output_path + "final_selection";
+    filesystem::create_directory(p);
+    p = output_path + "intermediate_selections";
+    filesystem::create_directory(p);
+    p = output_path + "alp";
+    filesystem::create_directory(p);
+  }
+
+  // N-1 selection
+  vector<const char*> selections = {
+    "l_{xy} > 200 #mum",
+    "|#eta^{#mu}| < 2.5",
+    "p_{T}^{#mu} > 10 GeV",
+    "m_{#mu#mu} #neq m_{J/#Psi},m_{#Psi}",
+  };
+
+  vector<int> color_palette = {
+    TColor::GetColor(230, 159, 0),
+    TColor::GetColor(86, 180, 233),
+    TColor::GetColor(0, 158, 115),
+    TColor::GetColor(0, 114, 178),
+    TColor::GetColor(213, 94, 0),
+  };
+  
+  map<string, tuple<int, string, int, bool, float, int, float> > file_names = {
+    // file name                                  // color  // legend            // linestyle    // signal   // cross-sec  // Ntot
+    {"01_muon_siblings/ttj.root",                  {33,    "t#bar{t}j resonant",            kSolid,    false,      395.3,        12540000, 1}},
```

Comment:
Oh wow, I had no idea `kSold` and `kDashed` exist, it was always 0, 1, 2, etc. for me. Good to know :)